study.md

```diff
@@ -57,5 +58,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+<!-- The user makes the request by clicking something like a button on the website which then goes to the router which formulates the proper request to one of the controllers. The controllers receive the url and formulate the GET request to the backend Model layer. This generates a SQL query that retrieved the desired data and passes it back trough the chain. -->
```

Comment:
Better to say the request triggers an action. Requests are turned into objects that are accessible within controllers, but typically the controller doesn't forward the request to the model. instead it uses information about the request to run a model method, and uses the return value from those methods to formulate a response.